spec/features/login_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :feature, ui: true do
+  let(:login_path) { '/en/session/new' }
+  let(:profile_path) { '/en/profile' }
+
+  before { visit login_path }
+
+  describe 'as a standard user' do
+    before { create :user }
+
+    before do
+      fill_inputs 'session', params.slice(:email, :password)
+    end
+
+    context 'with everything valid' do
+      let(:params) { attributes_for :user }
+
+      before { click_button 'commit' }
+
+      it('redirect to profile page') { expect(page).to have_current_path(profile_path) }
+      it { expect(page).to have_field('user_name', with: params[:name]) }
+      it { expect(page).to have_field('user_username', with: params[:username]) }
+      it { expect(page).to have_field('user_city', with: params[:city]) }
+      it { expect(page).to have_field('user_institution', with: params[:institution]) }
+      it { expect(page).to have_link('Ask for confirmation') }
+      it { expect(page).to have_no_link('Workers', visible: :all) }
```

Comment:
This is not a test of the login feature, but rather of the profile page. So it can be moved to a separate test group.

Reply:
So after login, we expect only redirect to profile?

Reply:
yes

Reply:
moved to profile spec